pkg/logstash.sysv

```diff
@@ -141,7 +141,6 @@ configtest() {
     return 1
   fi
 
-  JAVA_OPTS=${LS_JAVA_OPTS}
   HOME=${LS_HOME}
   export PATH HOME JAVA_OPTS LS_HEAP_SIZE LS_JAVA_OPTS LS_USE_GC_LOGGING
```

Comment:
We need to export here because if JAVA_OPTS isn't set at all, we set it here: https://github.com/elastic/logstash/blob/master/bin/logstash.lib.sh#L56 - and it not being set previously means we know it isn't exported as an environment variable, so we have to tell the shell to export it so that it is passed down to the JRuby executable - https://github.com/elastic/logstash/blob/master/bin/logstash.lib.sh#L181

It's possible that we should just export it on [this line](https://github.com/elastic/logstash/blob/master/bin/logstash.lib.sh#L66) when we're in the conditional knowing that JAVA_OPTS isn't set. But either way we get the correct behavior.